### ISSUE TITLE
identify.dendrogram bug when horiz = T

### ISSUE THREAD

**Issue by brooksambrose** [talgalili/dendextend#4]
The identify method doesn't appear to be working when `horiz=T`. This is my first bug report, sorry if this isn't the right venue! Below is an example taken from the dendextend tutorial. When clicking on the graphics device, `identify(*,horiz=T)` does not behave as expected, and it does not return a correct result.
```
require(dendextend)
require(colorspace)
data(iris)
d_iris <- dist(iris[,-5]) 
hc_iris <- hclust(d_iris)
dend_iris <- as.dendrogram(hc_iris)
iris_species <- rev(levels(iris[,5]))
dend_iris <- color_branches(dend_iris,k=3, groupLabels=iris_species)
labels_colors(dend_iris) <-rainbow_hcl(3)[sort_levels_values(as.numeric(iris[,5])[order.dendrogram(dend_iris)])]
labels(dend_iris) <- paste(as.character(iris[,5])[order.dendrogram(dend_iris)],"(",labels(dend_iris),")",sep = "")
dend_iris <- hang.dendrogram(dend_iris,hang_height=0.1)
dend_iris <- assign_values_to_leaves_nodePar(dend_iris, 0.5, "lab.cex")
par(mar = c(3,3,3,7))
plot(dend_iris,
main = "Clustered Iris dataset
(the labels give the true flower species)",
horiz = TRUE, nodePar = list(cex = .007))
legend("topleft", legend = iris_species, fill = rainbow_hcl(3))

identify(dend_iris,horiz=T)
```

**Comment by talgalili** [comment]
mmm, I only now noticed this - sorry for the late reply. It will take me some time to look at it, but I will. Thanks for the report!

**Comment by talgalili** [comment]
Hi brooksambrose,
Just to thank you again for this issue report. As you might have seen, it is now fixed in version 0.18.8.

Best,
Tal
